lesson7/L7HW.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+
+def check_name(db):
+    while True:
+        username = input("Enter your username for a registration.\n")
+        if username in db.keys():
+            print("This username is already taken.")
+        elif 10 < len(username) or 2 > len(username):
+            print("The name must consists of less than 10 chars and more than 1.")
+        else:
+            return username
+
+
+def get_password():
+    password = input("Create your password:\n")
+    while input("Repeat your password:\n") != password:
```

Comment:
it isnt the best design.. Try to communicate with [Mary](https://github.com/Feityanka),I am sure she can explain you why